prac2-pages/anshumali_suri/lib/main.dart

```diff
@@ -0,0 +1,202 @@
+import 'package:flutter/material.dart';
+
+void main() => runApp(const MyApp());
+
+class MyApp extends StatelessWidget {
+  const MyApp({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+
+      home: Scaffold(
+
+        body: const MyStatefulWidget(),
+
+      ),
+    );
+  }
+}
+
+class MyStatefulWidget extends StatefulWidget {
+  const MyStatefulWidget({Key? key}) : super(key: key);
+
+  @override
+  State<MyStatefulWidget> createState() => _MyStatefulWidgetState();
+}
+
+class _MyStatefulWidgetState extends State<MyStatefulWidget> {
+  TextEditingController nameController = TextEditingController();
+  TextEditingController passwordController = TextEditingController();
+  bool value=false;
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+        padding: const EdgeInsets.all(20),
+        child: ListView(
+          children: <Widget>[
+
+            Container(
+
+                alignment: Alignment.center,
+                padding: const EdgeInsets.all(10),
+                child: const Text(
+                  'Login',
+
+                  style: TextStyle(
+
+                      fontWeight: FontWeight.bold,
+                      fontSize: 30),
+                )),
+            Container(
+                alignment: Alignment.center,
+                padding: const EdgeInsets.all(10),
+                child:  Text(
+                  'Create an account',
+                  style: TextStyle(fontSize: 20,
+                    color: Colors.blueAccent,
+                  ),
+
+                )),
+            Container(
+              padding: const EdgeInsets.all(10),
```

Comment:
If you just wish to add padding in a container, use the Padding widget instead. 
Leads to a much more optimized code and improves the performance of your app.

Reply:
Resolved in commit [Changed Container widget to Padding](https://github.com/RitwikSingh28/GitHubPrac/pull/15/commits/4b8f5ce14d50ba439d243a747e119661daa05135)